Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -257,31 +239,20 @@ class DatabaseContainer: NSPersistentContainer {
         }
 
         writableContext.perform { [weak self] in
```

Comment:
does it makes sense to have it as a performAndWait?

Reply:
Yeah I thought about this as well 🤔 It could help make sure this is executed 👍

Reply:
Changed it 👍